src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -150,29 +150,38 @@ private static XmlSerializerNamespaces DefaultNamespaces
             }
         }
 
+        // Trimmer warning messages
+        internal const string TrimSerializationWarning = "Members from serialized types may be trimmed if not referenced directly";
+        private const string s_trimDeserializationWarning = "Members from deserialized types may be trimmed if not referenced directly";
```

Comment:
(nit) since this isn't static (it is const), it should be named `TrimDeserializationWarning`

Reply:
(nit: `const`s are static members according to the C# language specification 😄)

Reply:
Tell that to our https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

> 3. We use `_camelCase` for internal and private fields and use `readonly` where possible. Prefix internal and private instance fields with `_`, static fields with `s_`

> 12. We use PascalCasing to name all our constant local variables and fields.